rover_msgs/RoverLog.lcm

```diff
@@ -0,0 +1,12 @@
+package rover_msgs;
+
+struct RoverStatus {
+
+    uint8_t CRITICAL = 0; //Requires operator action. Provide basic recovery steps in <message>
+    uint8_t WARNING = 1; //Requires operator attention. Program should attempt self recovery
+    uint8_t DEBUG = 2; //Basic, useful information. No action necessary
+
+    uint8_t priority; //Set this to RoverStatus.CRITICAL, .WARNING, or .DEBUG

```

Comment:
Why did you choose to use a `uint8_t` (i.e. an enum) instead of a `string`? The nav status LCM used to do the same but we ended up having to store the enum definition in at least 3 locations. This meant when the possible values changed, you had to make sure to change all of them which is arguably not good style. On a less important note, this also requires users to memorize the mapping from number to meaning as the LCM echo as is won't translate it.

Reply:
With lcm structs you don't have to store the enum, the enum is stored in the lcm message's namespace. So you can say RoverLog.priority = RoverLog.CRITICAL.

Reply:
This method can't be done for JavaScript services (e.g. the base station), can it?